deploy/users-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  pgbouncer: bmV3X3Bhc3N3b3JkXzE=
+  postgres: bmV3X3Bhc3N3b3JkXzE=
+  primaryuser: bmV3X3Bhc3N3b3JkXzE=
+kind: Secret
```

Comment:
would user secrets creation be still enabled by default if `cluste1-users` is missing?

Reply:
secret will be generated automatically